src/fable-compiler-js/src/util.js

```diff
@@ -94,7 +107,7 @@ export function copyFolder(from, dest) {
 export function transformAndSaveBabelAst(babelAst, filePath, projDir, outDir, commonjs) {
     try {
         // this solves a weird commonjs issue where some imports are not properly qualified
-        babelAst = JSON.parse(JSON.stringify(babelAst)); // somehow this helps with that
+        babelAst = JSON.parse(serializeToJson(babelAst)); // somehow this helps with that
```

Comment:
Is this still needed? Does it work by removing this line?

Reply:
In what sense? This is the fix ...

Reply:
Yes, you're right. I'm trying several things but cannot make it work. I need to compare the ASTs before and after this line to see what's the issue. Thank you!